### DISCUSSION TITLE
Hoymiles HM-1800

### DISCUSSION THREAD

**Discussion by Schwatti** [tbnobody/OpenDTU#522] [Q&A] [answered]
Can I use OpenDTU with the Hoymiles HM-1800, or will it come on further updates?

**Comment by tbnobody** [comment]
I don't know any HM-1800 inverter. I only know the HMS-1800 which are using a completely different RF Module and frequency. If you really mean HM-1800 do you have some technical specs?

**Comment by Schwatti** [comment]
Yes, sorry, I mean the HMS-1800. I reread the specs and think that the sentence "The new Sub-1G wireless solution enables more stable communication with Hoymiles gateway DTU" gives me the information for another communication standard.

**Comment by tbnobody** [comment, marked as answer]
There are currently investigations (on the Discord server) with several RF modules. It seems that the protocol itself is nearly the same as for the HM models but due to the different hardware (for which currently no libraries exist) it will take some time to send and receive anything from the inverter. After this, the protocol implementation itself has to be adjusted/reverse engineered.

**Reply by Schwatti** [reply]
Thank you for the information.
